### PR TITLE
Add a comment to indicate that the authority needs to be configured when missing

### PR DESCRIPTION
**Pull request by javiercn**

### CODE REVIEW

File: src/ProjectTemplates/ComponentsWebAssembly.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Client/Pages/Index.razor [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 <h1>Hello, world!</h1>
 
+@*#if (MissingAuthority)
+<p>Before authentication will function correctly, you must configure your provider details in Program.cs</p>
```
Comment (SteveSandersonMS) [inline src/ProjectTemplates/ComponentsWebAssembly.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Client/Pages/Index.razor]:
```suggestion
<div class="alert alert-warning" role="alert">
    Before authentication will function correctly, you must configure your provider details in <code>Program.cs</code>
</div>
```

Comment (SteveSandersonMS) [reply src/ProjectTemplates/ComponentsWebAssembly.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Client/Pages/Index.razor]: I checked it - this markup works

File: src/ProjectTemplates/ComponentsWebAssembly.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Server/Program.cs [hunk 2]
```diff
@@ -22,6 +22,5 @@ public static void Main(string[] args)
                 {
                     webBuilder.UseStartup<Startup>();
                 });
```
Comment (SteveSandersonMS) [inline src/ProjectTemplates/ComponentsWebAssembly.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Server/Program.cs]: Can we also have a comment above `options.ProviderOptions.Authority = ...` saying something like:

```cs
// Configure your authentication provider options here
// For more information, see https://aka.ms/blazor-standalone-auth
```

I registered that fwlink just now.

Comment (SteveSandersonMS) [reply src/ProjectTemplates/ComponentsWebAssembly.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Server/Program.cs]: Only in the `MissingAuthority` case of course

Comment (javiercn) [reply src/ProjectTemplates/ComponentsWebAssembly.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Server/Program.cs]: Wrong program.cs :)

**Comment by javiercn** [comment]
image

**Review by SteveSandersonMS — APPROVED** [review]

**Pull request merged**